### PR TITLE
fix karmada-api folder docs for MDX3

### PR DESCRIPTION
**Pull request by Arhell**
**What type of PR is this?**

Add one of the following kinds:

/kind bug
/kind cleanup
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@RainbowMango

File: docs/reference/karmada-api/auto-scaling-resources/cron-federated-hpa-v1alpha1.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ CronFederatedHPAList contains a list of CronFederatedHPA.
 
 #### HTTP Request
 
-GET /apis/autoscaling.karmada.io/v1alpha1/namespaces/{namespace}/cronfederatedhpas/{name}
+GET /apis/autoscaling.karmada.io/v1alpha1/namespaces/\{namespace\}/cronfederatedhpas/\{name\}
```
Comment (RainbowMango) [inline docs/reference/karmada-api/auto-scaling-resources/cron-federated-hpa-v1alpha1.md:225]: Given tools generate these docs, I don't know if we can update them manually. 
cc tools author here for comments. @chaosi-zju

**Comment by Arhell** [comment]
@RainbowMango 
Please tell me which folders are generated, so that I can update those documents that are not generated and then think about what to do next.

**Comment by RainbowMango** [comment]
Files under `docs/reference` are almost auto-generated, except:
- docs/reference/instrumentation
- docs/reference/glossary.md
- docs/reference/object-association-mapping.md
- docs/reference/reserved-namespaces.md

**Pull request closed without merging**